applications/omnipresence/src/omnip_message_summary.erl

```diff
@@ -229,43 +220,7 @@ send_update(<<"amqp">>, _User, Props, Subscriptions) ->
                             )
          || S <- Stalkers
         ],
-    wh_amqp_worker:checkin_worker(Worker);
-send_update(<<"sip">>, User, Props, Subscriptions) ->
```

Comment:
maybe make it its own module if you want to clean up the code?

omnip_message_summary_amqp.erl
omnip_message_summary_nksip.erl

ect

Then use the new supervisor options to control with to start, supporting mix and match too ;)